README.md

```diff
@@ -0,0 +1,229 @@
+# Nushell RFCs
+
+[Nushell RFCs]: #nushell-rfcs
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Though some changes are substantial enough that we ask for these to be
+put through a bit of a design process and produce a consensus among the
+Nushell community.
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for new features to be added to Nushell, so that all
+stakeholders can be confident about the direction the project is evolving in.
+
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#nushell-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [What the process is]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [RFC Postponement]
+  - [License]
+  - [Contributions]
+  - [Acknowledgements]
+
+
+## When you should follow this process
+[When you should follow this process]: #when-you-should-follow-this-process
+
+*Note: While we establish this RFC process, this says "should"; eventually
+it should say "need to".*
+
+You should follow this process if you intend to make substantial changes to
+Nushell, or this RFC process itself. What constitutes a substantial change is
+evolving based on community norms and varies depending on what part of the
+ecosystem you are proposing to change, but may include the following:
+
+  - Any semantic or syntactic change that is not a bugfix.
+  - Removing existing features.
+  - Changes to internal interfaces, like the plugin architecture.
+
+Some changes do not require an RFC:
+
+  - Rephrasing, reorganizing, refactoring, or otherwise "changing shape does
+    not change meaning".
+  - Additions that strictly improve objective, numerical quality criteria
+    (warning removal, speedup, better platform coverage, more parallelism,
+    better handle errors, etc.)
+  - Additions only likely to be _noticed by_ other developers of Nushell,
+    invisible to users of Nushell.
```

Comment:
The language above doesn't scream "these lists aren't necessarily comprehensive" to me. I wonder if we could somehow be more explicit about that?

Reply:
I changed the line before the list to this: "Some changes do not require an RFC. For example:"

Reply:
👍 "For example" implies non-comprehensiveness.